doc/METADATA.ini

```
@@ -1283,4 +1283,4 @@ type = "enum
 	inlinetable"
 descriptions = "Tells the TOML plugin, which kind of TOML structure a key represents. When reading a TOML file, the plugin sets this metakey when encountering a TOML structure.
 	On writing, it is used for writing the structure back into a TOML file.
-	No metakey inference is done on writing, it's either set by a previous read of a TOML file or set explicitly by the user."
+	No metakey inference is done on writing, it's either set by a previous read of a TOML file or set explicitly by the user."
```

Any specific reason you removed the last newline?

```suggestion
	No metakey inference is done on writing, it's either set by a previous read of a TOML file or set explicitly by the user."

```

---

I had no specific reason, the last newline was automatically removed by the editor of the IDE.
I now changed the settings, so that newlines are not automatically added or removed.